### PR TITLE
docs app style updates

### LINKED ISSUE #14351: [mis-orgnized] different menus on same website: https://angularjs.org/
**Do you want to request a _feature_ or report a _bug_?**
feature

**What is the current behavior?**
I currently start to learn angualrjs on https://angularjs.org/, but the menu bar did make me misunderstand, and I feel mess about what I am going to learn. Could you make them uniform?

home1
home2

**What is the expected behavior?**
All pages with a same menu bar.

### LINKED ISSUE #14963: Logos out of date
https://github.com/angular/angular.js/tree/master/images/logo/AngularJS.exports

please update with highres versions with the 2015 google logo as seen on angularjs website

http://docs.angularjs.org/misc/faq

### PR DESCRIPTION
**Pull request by Narretz**
This updates the docs app to the new header style found on angularjs.org.

I've also taken the liberty to update the menu entries on the doc site - pending approval I'll update the org site, too (would fix https://github.com/angular/angular.js/issues/14351)

And fixes https://github.com/angular/angular.js/issues/14963

Notable differences between org / docs:
- search icon
- font family / weight (should we update?)
- no drop shadow on navbar because there's a secondary bar on docs

Notable differences between docs old / new:
- mobile: the top header + search are fixed, sub header is relative (previously all relative)
- mobile search close icon has moved over the search bar, because I had some problems giving it it's previous position while keeping the search results scrollable (oh my)

Implementation notes:
- angular-topnav.css is included as is from org site
- several overrides necessary because org uses bootstrap 2 / docs uses bs 3 (oh my)
- org site uses one breakpoint at 800px, docs uses one at 768 and one at 992. I tested it and it doesn't affect the common screen sizes

Future work:
- clean up the css and maybe implement sass for the docs app

Screens:

fireshot screen capture 014 - angularjs_ api_ api reference - localhost_8000_build_docs_api desktop dropdown open
fireshot screen capture 015 - angularjs_ api_ api reference - localhost_8000_build_docs_api desktop search
fireshot screen capture 016 - angularjs_ api_ api reference - localhost_8000_build_docs_api phone dropdown
fireshot screen capture 017 - angularjs_ api_ api reference - localhost_8000_build_docs_api phone search

### CODE REVIEW

File: docs/config/templates/app/indexPage.template.html [hunk 1]
```diff
@@ -68,101 +68,81 @@
...
+                </ul>
+              </li>
+              <li class="dropdown" uib-dropdown>
+                <a href="#" class="dropdown-toggle" uib-dropdown-toggle>Discuss</a>
+                <ul class="dropdown-menu" uib-dropdown-menu>
+                  <li><a href="http://blog.angularjs.org">Blog</a></li>
+                  <li><a href="http://groups.google.com/group/angular">Mailing List</a></li>
+                  <li><a href="http://webchat.freenode.net/?channels=angularjs&uio=d4">IRC</a></li>
+                  <li><a href="https://gitter.im/angular/angular.js">Gitter</a></li>
+                  <li><a href="https://twitter.com/#!/angular">Twitter</a></li>
```
Comment (gkalpak) [inline docs/config/templates/app/indexPage.template.html]: 👍 :smiley: (The `#!/` could be dropped: `https://twitter.com/angular`)

**Review by gkalpak — COMMENTED** [review]
Didn't serve it locally, but the code and the screenshots look very good :smiley:

**Comment by Narretz** [comment]
Tests have been fixed

**Review by IgorMinar — APPROVED** [review]
lgtm. please make the same menu change as with the angularjs.org menu (faq/tutorial swap).

thanks!

**Pull request merged**